### PR TITLE
add el8 instructions to plugins

### PR DESCRIPTION
**Pull request by evgeni**
also fix smart proxy plugin rpm names

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Just an idea, but on EL we have provides, so you can do `yum install foreman-plugin-discovery` on both EL7 and EL8. We do do the same on Debian now so it can't fully unify it.

The only downside is that `yum search foreman-plugin` doesn't do anything.

**Comment by evgeni** [comment]
I didn't want to start refactoring the whole page, and yes, the non-working search is rather annoying.

File: plugins/index.md [hunk 1]
```diff
@@ -265,6 +266,16 @@ gpgcheck=0
 </pre>
 </div>
 
+<div class="installation_os installation_os_el8">
+<pre>
+[foreman-plugins]
+name=Foreman plugins
+baseurl=https://yum.theforeman.org/plugins/{{page.version}}/el8/x86_64/
+enabled=1
+gpgcheck=0
```
Comment (ehelms) [inline plugins/index.md:275]: Ought we to put in `module_hotfixes=1` like our release RPM does to be on the safe side for plugins to work?

Comment (evgeni) [reply plugins/index.md:275]: We can, yeah

**Review by ehelms — APPROVED** [review]

**Pull request merged**